crmsh/ocfs2.py

```diff
@@ -299,6 +299,10 @@ def init_ocfs2(self):
             self._config_resource_stack_ocfs2_along()
         bootstrap.status("  OCFS2 device {} mounted on {}".format(self.target_device, self.mount_point))
 
+        res = utils.get_stdout_or_raise_error("crm configure get_property no-quorum-policy")
+        if res != "freeze":
+            utils.get_stdout_or_raise_error("crm configure property no-quorum-policy=freeze")
+
```

Comment:
@zzhou1 
> This setting is recommended for clusters where certain resources depend on communication with other nodes

What if these resources been deleted? We cannot change no-quorum-policy as stop back automatically

Reply:
I think it is fine. But, it worth to print a explict message to user the default value is changed to "freeze" here.